app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,103 @@
  *= require_tree .
  *= require_self
  */
+
+ *{
+    padding: 0;
+    margin: 0;
+    text-align : center;
+ }
```

Comment:
I think we can split this files into  multiple files for it to be more readable. We can have a reset, a global and a form file.